lib/commands/mixins.ts

```diff
@@ -11,6 +17,7 @@ import type {
 import type {EmptyObject} from 'type-fest';
 import {AndroidUiautomator2Driver} from '../driver';
 import type * as types from './types';
+import {Uiautomator2ExecuteMethodMap} from '../execute-method-map';
```

Comment:
is Uiautomator2ExecuteMethodMap used anywhere?

Reply:
nope